modules/apps/commerce/commerce-wish-list-api/src/main/java/com/liferay/commerce/wish/list/util/CommerceWishListThreadLocal.java

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
instead of this (that if I understand correctly is needed to avoid the "check" during the delete phase) evaluate what @andrea-ale-sbarra did for the CPConfigurationList where he created a `forceDelete` ?